test.js

```diff
@@ -186,3 +186,73 @@ test('remove keys (#36)', t => {
 	const actual = mapObject(object, mapper, {deep: true});
 	t.deepEqual(actual, expected);
 });
+
+test('mapper argument `path` is only available for deep mappings', t => {
```

Comment:
This is no longer correct.
```suggestion
test('mapper `path` argument', t => {
```